target/linux/ath79/dts/ar7161_adtran_bsap1800v2.dts

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "ar7161_adtran_bsap1880.dtsi"
```

Comment:
It's a little misleading that you derive 1840 and 1800 DTSes from a DTSI which has 1880. Consider using 18x0 there.

Reply:
So how would this board potentially be disambiguated from the BSAP-1800 v1, then?

Reply:
Don't know (There is no 1800v1 in ath79). Do we have to? What would be the difference?

Reply:
There was a v1 model which had an older "1800" board, in contrast to this "1880" board.

Reply:
Hmm, that's ugly. I still think the 1880 is misleading, but I'm short of a better idea. So keep it (also in the Makefile).
However, I would still do the v2 space/hyphen rename mentioned above.